tutorials/admin/getting-started/installation.md

```diff
@@ -1 +1,237 @@
+managing them yourself.
+
+```{note}
+If using [certbot](https://certbot.eff.org/), note that the Marble stack uses 
+[nginx](https://docs.nginx.com/nginx/admin-guide/web-server/reverse-proxy/) as a reverse proxy.
+
+Certbot also requires that the website be already running and that ports 80 and 443 are open.
+To deploy the stack and start the website see the instructions [here](deploy-marble).
+```
+
+### Git
+
+The Marble source code is hosted on [GitHub](https://github.com) and a `git` client is highly recommended to download 
+and update the source code.
+
+Your server may already have `git` installed. To check run the following command from the command line:
+
+```shell
+git --version
+```
+
+The output should be some information about the version of `git` currently installed on your machine.
+
+If you do not have `git` installed, it can be installed from https://git-scm.com/downloads. Select your operating system
+for information on how to install `git`.
+
+## Marble
+
+Marble is deployed using a piece of software named [Birdhouse](https://github.com/bird-house/birdhouse-deploy/).
+
+### Download the birdhouse software
+
+```shell
+git clone https://github.com/bird-house/birdhouse-deploy.git
+```
+
+This will install the [Birdhouse](https://github.com/bird-house/birdhouse-deploy/) source code to the current 
+directory.
+
+The most up-to-date version of the source code is on the `master` branch. We recommend always using this branch.
+
+### Create the local environment file
+
+Your Marble deployment can be configured and customized by editing a local environment file. To create this file:
+
+```shell
+cd birdhouse-deploy/birdhouse
+touch env.local
+```
+
+An (incomplete) example of this file named `env.local.example` can be used as a starting point. 
+
+```{warning}
+`env.local.example` contains a lot of default examples that *must* be updated. For this reason, we do not recommend 
+just copying `env.local.example` to `env.local`
+```
+
+### Configure `env.local`
+
+The `env.local` file contains environment variables that are used when deploying the Marble software.
+
+Instructions on which variables should be set in this file can be found in `env.local.example`. Please refer to this
+file which will instruct you:
+
+- which variables *must* be set for all deployments
+- which variables can be modified to customize your deployment
+- how to enable additional components and which variables to set to customize these components
+
+(deploy-marble)=
+## Deploying Marble
+
+The stack can be started, stopped and inspected using the `pavics-compose.sh` script which takes the same arguments as
+the [`docker compose`](https://docs.docker.com/compose/) commands.
+
+The `pavics-compose.sh` script will read the `env.local` configuration file, create a `docker compose` deployment file
+based on this configuration and then execute the subcommand based on that file. 
```

Comment:
The reference "the subcommands" seems undefined here since nothing preceding statements has clarified what are the subcommands.